src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,62 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/tab';
+
+type PropsTabs = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: FC<PropsTabs> = (props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const selectedTab = tabs
+    .find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const changeTab = (tab: Tab) => {
+    if (tab.id === selectedTab.id) {
+      return;
+    }
+
+    onTabSelected(tab);
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isSelected = selectedTab.id === tab.id;
```

Comment:
you use `isSelected` only one time, so let's pass this condition straight to classNames object. (remove `return` keyword in this case)

but you used the condition twice in this component, and if you want to leave `isSelected` variable, try to come up with some solution.